blocks/library/gallery/index.js

```diff
@@ -61,16 +70,11 @@ registerBlockType( 'core/gallery', {
 			query( 'div.blocks-gallery figure.blocks-gallery-image img', {
 				url: attr( 'src' ),
 				alt: attr( 'alt' ),
-			} ),
+			} ) || [],
```

Comment:
>@aduth wouldn't we need ES6 to make everything work anyway? ES5 would bring other complications, like class variables.

Not necessarily. What did you have in mind? Assigning `edit` and `save` as React class components could be challenging, but maybe reasonable at that point to use `create-react-class`.